### DISCUSSION TITLE
Flame Icon

### DISCUSSION THREAD

**Discussion by ploegy** [Blackymas/NSPanel_HA_Blueprint#484] [Q&A]
Hello,thanks for the great work,everything works well but i have one question regarding the "Flame" Icon.the 3rd icon next to the Relay indication.
in Version 3.1.0 it was possible to select the source.Now it seems to be bind to the Thermostat.
Is there a way to modify it manual? 
Andre

**Comment by Blackymas** [comment]
it has always been tied to the thermostat

**Reply by Ploegyde** [reply]
Yes,but in 3.1.0 i was able to select the entity  triggering the symbol.I was able to put my hot water on it and not the Heater.
Top Menu 04 (optional):
![image](https://user-images.githubusercontent.com/100301784/226548759-a5d7b0b8-f1aa-4a9c-8ea2-94551a3472f5.png)

**Reply by edwardtfn** [reply]
You can still assign it to a chip and will get to the same results, no?
